app/scripts/modules/core/src/managed/ArtifactDetail.tsx

```diff
@@ -350,7 +352,7 @@ export const ArtifactDetail = ({
               name={environmentName}
               resources={resourcesByEnvironment[environmentName]}
             >
-              <div className="sp-margin-l-right">
```

Comment:
slightly tangential to this PR but wasn't worth splitting out: this margin on the right side of constraint cards was from an earlier design iteration where the cards had rounded corners, and as a result looked better with some spacing on the right. The new designs with squared off corners look better flush with the right side of the container, which is how they are supposed to look per the mockups.